action_server/src/robocorp/action_server/_actions_import.py

```diff
@@ -189,7 +189,11 @@ def _add_actions_to_db(
             "Expected it to be 0.0.2 or higher"
         )
 
-    cmdline = [env.get("PYTHON_EXE", sys.executable), "-m", "robocorp.actions", "list"]
+    # FIXME: PYTHON_EXE will use robocorp.actions from holotree env.
+    #        Why? IMHO, we need to use the same python here as the action-server is using.
+    # python = env.get("PYTHON_EXE", sys.executable)
```

Comment:
@fabioz why are we using python from holotree here, not the same one the action-server is useing? Bug or I'm missing something?

Reply:
You're missing something. 

We can't use the python from the action server, we *must* use the python being used to run the target robot.

Consider the following: the user has a conda that depends on numpy and has a numpy import. In the action server we don't have the numpy dependency, so, if we try to use our python to load it we'll crash.

The proper way to develop here is create an environment, do `inv devinstall` (which will get the development version in the environment for all libraries) and develop based on that env.

Reply:
i.e.: this change must be reverted.